mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/Connection.cs

```diff
@@ -853,36 +854,228 @@ class PacketReader {
 				return res;
 			}
 
+			public int ReadFixedSize () {
+				if (connection.Version.AtLeast (2, 53)) {
+					bool isFixedSize = ReadByte () == 1;
+					int lenFixedSize = 1;
```

Comment:
This will increase the size of every value returned by the debugger by 1 byte, is this really needed ?

Reply:
The other solution is to create a new datatype like, MONO_TYPE_FIXED_SIZE, and then we read the datatype of fixed sized array later, if you think this is better, just let me know...and I change...

Reply:
I think that would be better, this patch is like 1000 lines just to support this rarely used feature.